fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -109,9 +110,9 @@ class OffloadConfig:
     """Class for specifying all arguments related to offloading parameters."""
 
     # Offload type: currently only supports: "ssd_offload"
-    offload_type: str = None
+    offload_type: Optional[str] = None
```

Comment:
Should we specify a default as well if we are making this optional?

Reply:
This is purely a mypy linter fix. Since we initialize it as None, and that doesn't match type (str) mypy complains.